enos/enos-modules.hcl

```diff
@@ -77,6 +87,19 @@ module "vault_cluster" {
   vault_install_dir = var.vault_install_dir
 }
 
+module "vault_cluster_ips" {
+  source = "./modules/vault_cluster_ips"
+
+  vault_install_dir = var.vault_install_dir
+}
+
+module "vault_unseal_nodes" {
```

Comment:
Can we add a comment and/or modify the name to indicate that this is a workaround for known issues? I wouldn't want future scenario authors to use it without understanding that we intend to remove it.

Reply:
The module itself is a required step in replication scenario, where the secondary nodes need to be unsealed if they have the seal type "Shamir".  The workaround for the issue is to call a custom script twice to unseal, instead of using the `vault_unseal_resource`.  So I added the comments with reported issue links within this module and not on here.